fresnel/interact.py

```diff
@@ -277,6 +279,9 @@ def _stop_rendering(self):
         self.rendering = False
 
     def _start_rendering(self):
+        # send signal
+        self.update_signal.emit(self._scene.camera)
```

Comment:
```suggestion
        self.rendering.emit(self._scene.camera)
```

Reply:
oh shoot! I'm sorry I didn't see this. I get flooded with a lot of github notifications that I'm not always sure if they're intended for me or a broader group I'm part of... Anyway, thanks for merging it!